project/MiMaFilters.scala

```diff
@@ -19,6 +19,9 @@ object MiMaFilters {
 
         ProblemFilters.exclude[DirectMissingMethodProblem]("scala.Conversion.underlying"),
         ProblemFilters.exclude[MissingClassProblem]("scala.Conversion$"),
+
+        ProblemFilters.exclude[ReversedMissingMethodProblem]("scala.quoted.Quotes#reflectModule#ValDefModule.let"),
+        ProblemFilters.exclude[DirectMissingMethodProblem]("scala.quoted.Quotes#reflectModule#ValDefModule.let"),
```

Comment:
Those won't do much here. Instead, put `ProblemFilters.exclude[ReversedMissingMethodProblem]("scala.quoted.Quotes#reflectModule#ValDefModule.let")` in `BackwardsBreakingChanges`, somewhere under the `// ReversedMissingMethodProblem`s are acceptable. See comment in `Breaking changes since last LTS` ` comment. Sorry for not being clear before